src/shim.js

```diff
@@ -1,5 +1,6 @@
 import webpage from "webpage";
 import system from "system";
+import {ensureFunctionBind} from "./shim_support";
```

Comment:
So I don't think you need to export a function. Just put the logic in the other file as you had before and doing import from "./other_file" should work.

I can also do it after my meeting.